package.json

```diff
@@ -12,7 +12,8 @@
     "test": "mocha --require @babel/register --preset babel --file test/setup.js 'test/**/*.spec.js'"
   },
   "dependencies": {
-    "bcrypt": "^4.0.0",
+    "axios": "^0.19.2",
+    "bcrypt": "^5.0.0",
```

Comment:
What was the reasoning for this major version bump?

Reply:
I was trying to fix that 'nan' npm error, someone on the internets mentioned upgrading bcrypt. Turned out it didn't fix it! I can roll this back if you want but also it seems like it shouldn't break anything?

Reply:
Nah go ahead and leave it, I was just curious.